src/cards/repo-card.js

```diff
@@ -76,78 +77,78 @@ const renderRepoCard = (repo, options = {}) => {
     </g>
   `;
 
-  const svgLanguage = primaryLanguage
-    ? `
+	const svgLanguage = primaryLanguage
+		? `
     <g data-testid="primary-lang" transform="translate(30, 0)">
       <circle data-testid="lang-color" cx="0" cy="-5" r="6" fill="${langColor}" />
       <text data-testid="lang-name" class="gray" x="15">${langName}</text>
     </g>
     `
-    : "";
+		: "";
 
-  const iconWithLabel = (icon, label, testid) => {
-    return `
+	const iconWithLabel = (icon, label, testid) => {
+		return `
       <svg class="icon" y="-12" viewBox="0 0 16 16" version="1.1" width="16" height="16">
         ${icon}
       </svg>
       <text data-testid="${testid}" class="gray" x="25">${label}</text>
     `;
-  };
-  const svgStars =
-    stargazers.totalCount > 0 &&
-    iconWithLabel(icons.star, totalStars, "stargazers");
-  const svgForks =
-    forkCount > 0 && iconWithLabel(icons.fork, totalForks, "forkcount");
-
-  const starAndForkCount = FlexLayout({
-    items: [svgStars, svgForks],
-    gap: 65,
-  }).join("");
-
-  const card = new Card({
-    title: header,
-    titlePrefixIcon: icons.contribs,
-    width: 400,
-    height,
-    colors: {
-      titleColor,
-      textColor,
-      iconColor,
-      bgColor,
-    },
-  });
-
-  card.disableAnimations();
-  card.setHideBorder(false);
-  card.setHideTitle(false);
-  card.setCSS(`
+	};
+	const svgStars =
+		stargazers.totalCount > 0 &&
+		iconWithLabel(icons.star, totalStars, "stargazers");
+	const svgForks =
+		forkCount > 0 && iconWithLabel(icons.fork, totalForks, "forkcount");
+
+	const starAndForkCount = FlexLayout({
+		items: [svgStars, svgForks],
+		gap: 65
+	}).join("");
+
+	const card = new Card({
+		title: header,
+		titlePrefixIcon: icons.contribs,
+		width: 400,
+		height,
+		colors: {
+			titleColor,
+			textColor,
+			iconColor,
+			bgColor
+		}
+	});
+
```

Comment:
Please make sure nothing else changes other than the code you want. all the code is messed up. make sure your codeeditor does not have wrong tabs/space configs

Reply:
> Please make sure nothing else changes other than the code you want. all the code is messed up. make sure your codeeditor does not have wrong tabs/space configs

What are the settings you use then?